jss-old-data/src/main/java/org/bahmni/jss/JSSMigrator.java

```diff
@@ -64,8 +65,8 @@ public static void main(String[] args) throws URISyntaxException, IOException, C
     }
 
     private static void logPropertyUsage(String openMRSHostName, String databaseUserId, String databaseUserPassword, String openmrsUserId, String openmrsPassword) {
-        logger.info(String.format("By default uses following properties: openmrs.host.name=%s; database.user.id=%s; database.user.password=%s; openmrs.user.id=%s; " +
-                "openmrs.user.password=%s", openMRSHostName, databaseUserId, databaseUserPassword, openmrsUserId, openmrsPassword));
+        logger.info(String.format("By default uses following properties: openmrs.host.name=%s; database.user.id=%s; database.user.password=%s; openmrs.user.id=%s; openmrs.user.password=%s",
+                openMRSHostName, databaseUserId, databaseUserPassword, openmrsUserId, openmrsPassword));
```

Comment:
why log passwords?

Reply:
you can also use log4j formatter for logging instead of String.format... 
```
logger.printf(Level.INFO, "Logging in user %1$s with birthday %2$tm %2$te,%2$tY", user.getName(), user.getBirthdayCalendar());
```

Reply:
This code is no longer being used. Best to delete the password logging and commit it.